highfidelity/hifi
# Fixed null file path return (specific to music visualizer)

Test plan:

1. Download music visualizer app: https://hifi-content.s3.amazonaws.com/elisalj/music_visualizer/musicVisualizer.js and test sound: https://hifi-content.s3.amazonaws.com/elisalj/music_visualizer/test_sounds/UpbeatFunk_mono.wav
2. Open the app and click "Choose Audio File"
3. Select "Upbeat Funk"
4. Click on any particle, you should hear the music and the particle should "dance"
